src/ResourceManager/Insights/Commands.Insights/Alerts/AddAzureRmLogAlertRuleCommand.cs

```diff
@@ -91,6 +91,7 @@ private RuleCondition CreateRuleCondition()
         protected override AlertRuleResource CreateSdkCallParameters()
         {
             WriteWarning("*** Add-AzureRMLogAlertRule cmdlet will be deprecated in a future release.");
+            WriteWarning("*** Note: After October 1st using this cmdlet will no longer have any effect as this functionality is being transitioned to Activity Log Alerts. Please see https://aka.ms/migratemealerts for more information.");
```

Comment:
This cmdlet will be remvoed in the 5.0 release (November).  Please add this as appropriate to the deprecation comments throughout.